### PR TITLE
amascia/iac modify scan output message and style

### PR DESCRIPTION
**Pull request by amascia-gg**
Three changes:
- changed colour of policy title in output message
- added policy GG ID in output message
- click raises error when file is given to iac scan command

### CODE REVIEW

File: ggshield/cmd/iac/scan.py [hunk 1]
```diff
@@ -78,6 +79,7 @@ def scan_cmd(
     update_context(
         ctx, exit_zero, minimum_severity, verbose, ignore_policies, ignore_paths, json
     )
+
```
Comment (gg-mmill) [inline ggshield/cmd/iac/scan.py:82]: Nit: avoid commiting meaningless changes.

File: ggshield/output/text/message.py [hunk 2]
```diff
@@ -222,7 +222,9 @@ def iac_vulnerability_header(issue_n: int, vulnerability: IaCVulnerability) -> s
         issue_n,
         format_text("IaC", STYLE["detector"]),
         format_text(vulnerability.component, STYLE["detector"]),
-        format_text(vulnerability.policy, STYLE["detector"]),
+        format_text(
```
Comment (gg-mmill) [inline ggshield/output/text/message.py:225]: I think it'd be better to change the format  template: add ` ({})` to the templatre, as well as an argument dedicated to policy_id (we might want to change the color of the id only, plus having white parenthesis allow to make it stand out a bit more.

File: ggshield/cmd/iac/scan.py (hunk 1 above)
Comment (amascia-gg) [reply ggshield/cmd/iac/scan.py]: Yes sorry, the PR was not ready...

File: ggshield/output/text/message.py (hunk 2 above)
Comment (amascia-gg) [reply ggshield/output/text/message.py]: Much better like this indeed, thanks

**Review by gg-mmill — APPROVED** [review]
LGTM !

**Pull request merged**